topics/variant-analysis/tutorials/sars-cov-2-variant-discovery/tutorial.md

```diff
@@ -0,0 +1,750 @@
+   > >    - {% icon param-file %} *"Filter*": `Combined Variant Report by Variant`
+   > >    - *"With following condition*": `c1=='ERR5931005'` (to adapt with the sample name)
+   > >    - *"Number of header lines to skip*": `1`
+   > >
+   > >    We got then only the AVs for the selected sample (46 for ERR5931005).
+   > >
+   > > 3. To get the number of AVs for each sample, we can run {% tool [Group data](Grouping1) %} with the following parameters:
+   > >    - {% icon param-file %} *"Select data"*: `Combined Variant Report by Variant`
+   > >    - *"Group by column"*: `Column: 1`
+   > >    - In *"Operation"*:
+   > >      - In *"1: Operation"*:
+   > >        - *"Type"*: `Count`
+   > >        - *"On column"*: `Column: 2`
+   > >    
+   > >    With our example datasets, it seems that samples have between 41 and 56 AVs. 
+   > {: .solution}
+   {: .question}
+
+2. **Combined Variant Report by Variant**: this table combine the information (column) for each AV (row)
+
+   Column | Field | Meaning
+   --- | --- | ---
+   1 | `POS` | Position in [NC_045512.2](https://www.ncbi.nlm.nih.gov/nuccore/1798174254)
+   2 | `REF` | Reference base
+   3 | `ALT` | Alternative base
+   4 | `IMPACT` | Functional impact (from SNPEff) 
+   5 | `FUNCLASS` | Funclass for change (from SNPEff)
+   6 | `EFFECT` | Effect of change (from SNPEff)
+   7 | `GENE` | Gene 
+   8 | `CODON` | Codon 
+   9 | `AA` | Amino acid 
+   10 |`TRID` | Short name for the gene 
+   11 |`countunique(Sample)` | Number of distinct samples containing this change 
+   12 |`min(AF)` | Minimum Alternative Allele Freq across all samples containing this change 
+   13 |`max(AF)` | Maximum Alternative Allele Freq across all samples containing this change 
+   14 |`SAMPLES(above-thresholds)` | List of distinct samples where this change has frequency abobe threshold (5%)
+   15 |`SAMPLES(all)` | List of distinct samples containing this change at any frequency (including below threshold) 
+   16 |`AFs(all)` | List of all allele frequencies across all samples 
+   17 |`change` |  Change 
+
+   > ### {% icon question %} Questions
+   > 
+   > 1. How many AVs are found?
+   > 1. What are the different impacts of the AVs ?
+   > 2. How many variants are found for each impact?
+   > 3. What are the different effects for HIGH impact?
+   > 4. Is there any AVs impacting all samples?
+   >
+   > > ### {% icon solution %} Solution
+   > >
+   > > 1. By expanding the dataset in the history, we have the number of lines in the file. 191 lines for the example datasets. The first line is the header of the table. Then 190 AVs.
+   > >
+   > > 2. The different impacts of the AVs are HIGH, MODERATE and LOW.
+   > >
+   > > 2. To get the number of AVs for each impact levels, we can run {% tool [Group data](Grouping1) %} with the following parameters:
```

Comment:
and here 2 is twice

Reply:
It should not matter. Markdown resolves it :smile: